fix: correctly handle pathnames found in the `_redirects` file (#12821)
sveltejs/kit: documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

- `exclude` defines routes that will _not_ invoke a function — this is a faster and cheaper way to serve your app's static assets. This array can include the following special values:
	- `<build>` contains your app's build artifacts (the files generated by Vite)
	- `<files>` contains the contents of your `static` directory

> 	- `<redirects>` contains a list of pathnames from your [`_redirects` file](https://developers.cloudflare.com/pages/configuration/redirects/) in the `static` directory

Reviewer: I think this needs updating due to the major bump
```suggestion
	- `<redirects>` contains a list of pathnames from your [`_redirects` file](https://developers.cloudflare.com/pages/configuration/redirects/) at the root
```
